Content.Server/Chat/EmoteOnDamageComponent.cs

```diff
@@ -24,6 +24,19 @@ public sealed class EmoteOnDamageComponent : Component
     [DataField("emotes", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<EmotePrototype>)), ViewVariables(VVAccess.ReadWrite)]
     public HashSet<string> Emotes = new();
 
+    /// <summary>
+    /// Also send the emote in chat.
+    /// <summary>
+    [DataField("withChat"), ViewVariables(VVAccess.ReadWrite)]
+    public bool WithChat = false;
+
+    /// <summary>
+    /// Hide the chat message from the chat window, only showing the popup.
+    /// This does nothing if WithChat is false.
+    /// <summary>
+    [DataField("hiddenFromChatWindow")]
+    public bool HiddenFromChatWindow = false;
```

Comment:
arent these the same or am i missing something? (also dont forget the video in the pr body)